examples/with-apollo-auth/package.json

```diff
@@ -9,6 +9,8 @@
     "start": "next start"
   },
   "dependencies": {
+    "@apollo/react-hooks": "3.0.1",
```

Comment:
Did you miss ^ or it is on purpose?

Reply:
I just installed via `yarn add @apollo/react-hooks`. Didn't do anything special with it. And in my opinion sometimes it's better to lock concrete version of the package, especially for examples. So it will be safe for end-user to install deps.

Reply:
You right, but if we did not lock it (especially for the examples that developers use it regularly) users may report some issues with it, so we can make sure it works overtime.

Reply:
I locked it. End user will always get version 3.0.1 and nothing else, but anyway we can with carets